connectors/agent/protocol.py

```diff
@@ -0,0 +1,72 @@
+from elastic_agent_client.generated import elastic_agent_client_pb2 as proto
+from elastic_agent_client.handler.action import BaseActionHandler
+from elastic_agent_client.handler.checkin import BaseCheckinHandler
+from elastic_agent_client.util.logger import logger
+
+
+class ConnectorActionHandler(BaseActionHandler):
```

Comment:
Super nit/opinion - I don't like having multiple classes in one file like this, since I can never remember file names. I'd rather see each of these in their own file.